src/main/java/oscar/oscarMDS/data/CategoryData.java

```diff
@@ -426,16 +454,31 @@ public int getHRMDocumentCountForPatient() throws SQLException{
 		int count = 0;
 		PatientInfo info;
 
-		String sql = " SELECT HIGH_PRIORITY demographic_no, first_name, last_name, COUNT( distinct h.id) as count "
-						+ " FROM HRMDocument h "
-						+ " LEFT JOIN HRMDocumentToDemographic hd ON h.id = hd.hrmDocumentId"
-						+ " LEFT JOIN HRMDocumentToProvider hp ON h.id = hp.hrmDocumentId"
-						+ " LEFT JOIN demographic d ON hd.demographicNo = d.demographic_no"
-						+ " WHERE h.id IN (SELECT hrmDocumentId FROM HRMDocumentToDemographic hd)"
+		// String sql = " SELECT HIGH_PRIORITY demographic_no, first_name, last_name, COUNT( distinct h.id) as count "
```

Comment:
This suggestion isn't worth pursuing as it won't improve performance. Rather than focusing on removing code duplications, the real solution would be to refactor CategoryData.java to use JPQL for SQL queries, but that falls outside the scope of the current work.